package.json

```diff
@@ -23,7 +23,8 @@
     "@theforeman/vendor": "^4.3.0",
     "intl": "~1.2.5",
     "jed": "^1.1.1",
-    "react-intl": "^2.8.0"
+    "react-intl": "^2.8.0",
+    "react-markdown": "^4.3.1"
```

Comment:
can you please add this dependency to `@theforeman/vendor`?
https://github.com/theforeman/foreman-js/tree/master/packages/vendor-core

Reply:
@amirfefer I have created the [PR](https://github.com/theforeman/foreman-js/pull/140) in foreman-js but Travis is failing due to something. 
Can you help? Do I also need to upload `package-lock.json`?

Reply:
I see that the PR is merged :+1: 
can you please bump `foreman-js` sub-packages to version 4.7.0 and remove `react-markdown`?